python/lsst/meas/base/references.py

```diff
@@ -168,7 +188,7 @@ def fetchInPatches(self, dataRef, patchList):
 
         The given dataRef must include the tract in its dataId.
         """
-        dataset = self.config.coaddName + "Coadd_src"
+        dataset = self.config.coaddName + "Coadd_" + self.datasetSuffix
```

Comment:
Once we get to three terms in a string concatenation, I think it's probably time to use e.g. `"{}Coadd_{}".format(self.config.coaddName, self.datasetSuffix)` (or `%`, but I'm not sure that's compatible with Python 3).